config/environment.js

```diff
@@ -151,6 +153,9 @@ module.exports = function(environment) {
       id: 467005,
       enabled: environment === 'production',
       forceSSL: true
+    },
+    alienDom: {
+      keepInDom: `${ALIEN_DOM_ROOT} > :not(.ember-view):not(#fb-root), ${ALIEN_DOM_ROOT} > head > link[rel=stylesheet]:not([href*=assets])`
```

Comment:
Isn't this list of selectors for what we want to remove, not keep?

It's always going to be a little confusing with the double negative, (we _don't_ want nodes that are :_not_ x), and I don't think there's much we can do about that, but I don't think calling this 'keep' is helping.